crates/core/tedge_mapper/src/c8y/tests.rs

```diff
@@ -459,6 +462,10 @@ async fn c8y_mapper_syncs_pending_alarms_on_startup() {
     //     .await
     //     .unwrap();
 
+    // wait for session to get closed properly.
+    // Note: Need to find a better way to clean the session in the broker
+    // tokio::time::sleep(Duration::from_secs(20)).await;
+    clear_session().await;
```

Comment:
The comment is puzzling. What is the point of clearing the session? Do we still have to find a better alternative.

Reply:
removed